app/mm-redux/actions/search.ts

```diff
@@ -213,7 +211,7 @@ export function getRecentMentions(): ActionFunc {
             const terms = termKeys.map(({key}) => key).join(' ').trim() + ' ';
 
             analytics.trackAPI('api_posts_search_mention');
-            posts = await Client4.searchPosts(teamId, terms, true);
+            posts = await Client4.searchPosts('', terms, true);
```

Comment:
Just a question for my own understanding - by not passing in the teamId, you are getting the searchPosts results across all teams for these terms ?

Reply:
Yes, that is the plan.